src/plugins/chart_expressions/expression_xy/public/expression_renderers/xy_chart_renderer.tsx

```diff
@@ -57,6 +56,7 @@ export const getXyChartRenderer = ({
     const deps = await getStartDeps();
 
     const { XYChartReportable } = await import('../components/xy_chart');
+    const { calculateMinInterval } = await import('../helpers/interval');
```

Comment:
This will do the async loading in series - use `Promise.all` to parallelize

Reply:
Done.